### DISCUSSION TITLE
URGENT: crazy loud alarm overriding sound file AND sound level!

### DISCUSSION THREAD

**Discussion by fares57** [NightscoutFoundation/xDrip#2616] [Q&A]
Hello all! This is rather urgent. I have a poco F2 pro and suddenly my alarms are overriding all their settings either from xdrip or system and they sound with a new file sound which is not the one I chose but the worst thing is that they sound at the highest possible level so that all the city can hear it in the middle of the night. I can do nothing to change it. I updated the APK and restarted and it happened after a system update of poco!

I need help with this!

**Comment by Navid200** [comment]
Please check the volume profile xDrip is set to and change if you need to:
https://navid200.github.io/xDrip/docs/Volume-profiles.html

Each alert allows you to choose a sound file.  https://navid200.github.io/xDrip/docs/Custom-Alert-Sound.html

**Reply by fares57** [reply]
thank you but this is irrelevant unfortunately. I have it on HIGH in another phone and still the volume is quite reasonable and adjustable from the phone settings. It must have been something else that changed or glitched.

**Reply by Navid200** [reply]
If the alert is a glucose level alert, the phone volume will have no impact.
Only the volume profile will.

For all the other alerts, the volume that affects the alert is the ring and notification volume.  The volume profile in that case makes no difference and all the other volumes on the phone also make no difference.

**Reply by fares57** [reply]
After a more careful observation I realised that the volume was indeed the same. But though the alert settings had the bubble sound, as always, in reality the alert played a different alarm much more continuing and dramatic!! I was unable to fix it. It is clearly a problem of the app, in my opinion caused by a poco update. I settled for changing the sound again to a new one and then the app accept the setting and the alert played indeed the sound I chose (which was a very short whistle sound so that the loud volume wouldn't disturb the whole town in the night!)
